CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `flatten_dimensions`
     - `load_ml_model`
     - `predict_random_forest`
+    - `resample_vector`
```

Comment:
Sorry if I missed the discussion about this somewhere but what about naming this `reproject_vector`. Without knowing what the process does I assumed from the name something else (that it would modify the data - aggregate it but it seems to "only" change the CRS).

Reply:
I agree with @dthiex, the name does not seem to fit, since there is no resampling involved but just a reprojection of the points composing the vector data.

Reply:
I was just taking the prefix from resample_spatial and resample_cube_temporal etc as it seems to be a group of similar processes. It changes the CRS, it may change the labels, but not the actual data. I have no strong preference either way though.

Reply:
Rename to ` vector_reproject`.